### ISSUE TITLE
[java-spanner-jdbc] Dependency Dashboard

### ISSUE THREAD

**Issue by forking-renovate[bot]** [googleapis/google-cloud-java#12286]
This issue lists Renovate updates and detected dependencies. Read the [Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) docs to learn more.<br>[View this repository on the Mend.io Web Portal](https://developer.mend.io/github/googleapis/java-spanner-jdbc).

## Config Migration Needed

 - [ ]  Select this checkbox to let Renovate create an automated Config Migration PR.

## Rate-Limited

The following updates are currently rate-limited. To force their creation now, click on a checkbox below.

 - [ ] deps: update dependency org.springframework.boot:spring-boot-starter-data-jdbc to v4.0.5
 - [ ] deps: update dependency org.springframework.boot:spring-boot-starter-parent to v3.5.13
 - [ ] deps: update dependency org.testcontainers:testcontainers to v2.0.4
 - [ ] deps: update dependency org.testcontainers:testcontainers-bom to v2.0.4
 - [ ] deps: update dependency com.google.api.grpc:proto-google-cloud-trace-v1 to v2.88.0
 - [ ] deps: update dependency com.google.cloud:google-cloud-trace to v2.88.0
 - [ ] deps: update dependency com.google.cloud:sdk-platform-java-config to v3.58.0
 - [ ] 🔐 **Create all rate-limited PRs at once** 🔐

## Open

The following updates have all been created. To force a retry/rebase of any, click on a checkbox below.

 - [ ] [deps: update dependency com.fasterxml.jackson.core:jackson-databind to v2.21.2](../pull/2427)
 - [ ] [deps: update dependency org.springframework.boot:spring-boot to v4.0.5](../pull/2428)
 - [ ] **Click on this checkbox to rebase all open PRs at once**

## PR Closed (Blocked)

The following updates are blocked by an existing closed PR. To recreate the PR, click on a checkbox below.

 - [ ] [build(deps): update dependency org.apache.maven.plugins:maven-shade-plugin to v3.6.2](../pull/2417)
 - [ ] [chore(deps): update dependency com.google.cloud:google-cloud-spanner-jdbc to v2.35.5](../pull/2415)
 - [ ] [deps: update dependency net.bytebuddy:byte-buddy to v1.18.7-jdk5](../pull/2412)
 - [ ] [deps: update dependency net.bytebuddy:byte-buddy-agent to v1.18.7-jdk5](../pull/2413)
 - [ ] [chore(deps): update dependency com.google.cloud:libraries-bom to v26.78.0](../pull/2416)
 - [ ] [deps: update dependency com.google.cloud:google-cloud-spanner to v6.113.0](../pull/2422)
 - [ ] [deps: update dependency io.opentelemetry:opentelemetry-bom to v1.60.1](../pull/2418)
 - [ ] [deps: update dependency net.datafaker:datafaker to v1.9.0](../pull/1784)
 - [ ] [deps: update dependency org.springframework.data:spring-data-bom to v2025.1.4](../pull/2287)
 - [ ] [deps: update dependency net.datafaker:datafaker to v2](../pull/1785)
 - [ ] [deps: update dependency org.mybatis.dynamic-sql:mybatis-dynamic-sql to v2](../pull/2419)
 - [ ] [deps: update dependency org.mybatis.spring.boot:mybatis-spring-boot-starter to v4](../pull/2309)
 - [ ] [deps: update dependency org.springframework.boot:spring-boot-starter-parent to v4](../pull/2302)
 - [ ] [test(deps): update dependency org.mockito:mockito-core to v5](../pull/1101)

## Detected Dependencies

<details><summary>github-actions (6)</summary>
<blockquote>

<details><summary>.github/workflows/integration-tests-against-emulator.yaml (2)</summary>

 - `actions/checkout v6`
 - `actions/setup-java v5`

</details>

<details><summary>.github/workflows/quickperf.yaml (2)</summary>

 - `actions/checkout v6`
 - `actions/setup-java v5`

</details>

<details><summary>.github/workflows/sample-tests.yml (2)</summary>

 - `actions/checkout v6`
 - `actions/setup-java v5`

</details>

<details><summary>.github/workflows/spring-data-jdbc-sample.yaml (2)</summary>

 - `actions/checkout v6`
 - `actions/setup-java v5`

</details>

<details><summary>.github/workflows/spring-data-mybatis-sample.yaml (2)</summary>

 - `actions/checkout v6`
 - `actions/setup-java v5`

</details>

<details><summary>.github/workflows/unmanaged_dependency_check.yaml (3)</summary>

 - `actions/checkout v6`
 - `actions/setup-java v5`
 - `googleapis/sdk-platform-java google-cloud-shared-dependencies/v3.57.0`

</details>

</blockquote>
</details>

<details><summary>maven (10)</summary>
<blockquote>

<details><summary>pom.xml (15)</summary>

 - `com.google.cloud:sdk-platform-java-config 3.58.0`
 - `com.google.cloud:google-cloud-spanner-bom 6.113.0`
 - `io.opencensus:opencensus-api 0.31.1`
 - `io.opencensus:opencensus-contrib-grpc-util 0.31.1`
 - `org.testcontainers:testcontainers 2.0.3` → [Updates: `2.0.4`]
 - `com.google.truth:truth 1.4.5`
 - `org.mockito:mockito-core 4.11.0` → [Updates: `5.23.0`]
 - `junit:junit 4.13.2`
 - `com.google.cloud.opentelemetry:exporter-trace 0.36.0`
 - `io.opentelemetry:opentelemetry-api-incubator 1.45.0-alpha`
 - `com.google.cloud:google-cloud-trace 2.86.0` → [Updates: `2.88.0`]
 - `com.google.api.grpc:proto-google-cloud-trace-v1 2.86.0` → [Updates: `2.88.0`]
 - `org.apache.maven.plugins:maven-shade-plugin 3.6.1` → [Updates: `3.6.2`]
 - `org.apache.maven.plugins:maven-shade-plugin 3.6.1` → [Updates: `3.6.2`]
 - `org.apache.maven.plugins:maven-project-info-reports-plugin 3.9.0`

</details>

<details><summary>samples/install-without-bom/pom.xml (5)</summary>

 - `com.google.cloud.samples:shared-configuration 1.2.2`
 - `com.google.cloud:google-cloud-spanner-jdbc 2.35.3` → [Updates: `2.35.5`]
 - `junit:junit 4.13.2`
 - `com.google.truth:truth 1.4.5`
 - `org.codehaus.mojo:build-helper-maven-plugin 3.6.1`

</details>

<details><summary>samples/pom.xml (3)</summary>

 - `com.google.cloud.samples:shared-configuration 1.2.2`
 - `org.apache.maven.plugins:maven-deploy-plugin 3.1.4`
 - `org.sonatype.plugins:nexus-staging-maven-plugin 1.7.0`

</details>

<details><summary>samples/quickperf/pom.xml (10)</summary>

 - `com.google.cloud:sdk-platform-java-config 3.57.0` → [Updates: `3.58.0`]
 - `com.google.cloud:libraries-bom 26.76.0` → [Updates: `26.78.0`]
 - `net.datafaker:datafaker 1.7.0` → [Updates: `1.9.0`, `2.5.4`]
 - `commons-cli:commons-cli 1.11.0`
 - `org.apache.commons:commons-lang3 3.20.0`
 - `com.fasterxml.jackson.core:jackson-databind 2.21.1` → [Updates: `2.21.2`]
 - `org.testcontainers:testcontainers 2.0.3` → [Updates: `2.0.4`]
 - `org.springframework.boot:spring-boot 4.0.3` → [Updates: `4.0.5`]
 - `junit:junit 4.13.2`
 - `org.codehaus.mojo:exec-maven-plugin 3.6.3`

</details>

<details><summary>samples/snapshot/pom.xml (5)</summary>

 - `com.google.cloud.samples:shared-configuration 1.2.2`
 - `com.google.cloud:google-cloud-spanner-jdbc 2.35.5`
 - `junit:junit 4.13.2`
 - `com.google.truth:truth 1.4.5`
 - `org.codehaus.mojo:build-helper-maven-plugin 3.6.1`

</details>

<details><summary>samples/snippets/pom.xml (4)</summary>

 - `com.google.cloud:sdk-platform-java-config 3.57.0` → [Updates: `3.58.0`]
 - `com.google.cloud:libraries-bom 26.76.0` → [Updates: `26.78.0`]
 - `org.testcontainers:testcontainers 2.0.3` → [Updates: `2.0.4`]
 - `junit:junit 4.13.2`

</details>

<details><summary>samples/spring-data-jdbc/googlesql/pom.xml (11)</summary>

 - `org.springframework.data:spring-data-bom 2025.0.5` → [Updates: `2025.1.4`]
 - `com.google.cloud:google-cloud-spanner-bom 6.113.0`
 - `com.google.cloud:libraries-bom 26.76.0` → [Updates: `26.78.0`]
 - `io.opentelemetry:opentelemetry-bom 1.59.0` → [Updates: `1.60.1`]
 - `org.springframework.boot:spring-boot-starter-data-jdbc 4.0.3` → [Updates: `4.0.5`]
 - `com.google.cloud.opentelemetry:exporter-trace 0.36.0`
 - `com.google.cloud.opentelemetry:exporter-metrics 0.36.0`
 - `org.testcontainers:testcontainers 2.0.3` → [Updates: `2.0.4`]
 - `com.google.collections:google-collections 1.0`
 - `junit:junit 4.13.2`
 - `com.spotify.fmt:fmt-maven-plugin 2.29`

</details>

<details><summary>samples/spring-data-jdbc/postgresql/pom.xml (14)</summary>

 - `org.springframework.data:spring-data-bom 2025.0.5` → [Updates: `2025.1.4`]
 - `com.google.cloud:google-cloud-spanner-bom 6.113.0`
 - `com.google.cloud:libraries-bom 26.76.0` → [Updates: `26.78.0`]
 - `io.opentelemetry:opentelemetry-bom 1.59.0` → [Updates: `1.60.1`]
 - `org.springframework.boot:spring-boot-starter-data-jdbc 4.0.3` → [Updates: `4.0.5`]
 - `org.postgresql:postgresql 42.7.10`
 - `com.google.cloud.opentelemetry:exporter-trace 0.36.0`
 - `com.google.cloud.opentelemetry:exporter-metrics 0.36.0`
 - `com.google.collections:google-collections 1.0`
 - `net.bytebuddy:byte-buddy 1.18.5` → [Updates: `1.18.7-jdk5`]
 - `net.bytebuddy:byte-buddy-agent 1.18.5` → [Updates: `1.18.7-jdk5`]
 - `junit:junit 4.13.2`
 - `org.testcontainers:testcontainers 2.0.3` → [Updates: `2.0.4`]
 - `com.spotify.fmt:fmt-maven-plugin 2.29`

</details>

<details><summary>samples/spring-data-mybatis/googlesql/pom.xml (13)</summary>

 - `org.springframework.boot:spring-boot-starter-parent 3.5.11` → [Updates: `3.5.13`, `4.0.5`]
 - `org.springframework.data:spring-data-bom 2025.0.5` → [Updates: `2025.1.4`]
 - `com.google.cloud:google-cloud-spanner-bom 6.113.0`
 - `com.google.cloud:libraries-bom 26.76.0` → [Updates: `26.78.0`]
 - `org.testcontainers:testcontainers-bom 2.0.3` → [Updates: `2.0.4`]
 - `org.mybatis.spring.boot:mybatis-spring-boot-starter 3.0.5` → [Updates: `4.0.1`]
 - `org.mybatis.dynamic-sql:mybatis-dynamic-sql 1.5.2` → [Updates: `2.0.0`]
 - `com.google.collections:google-collections 1.0`
 - `com.google.cloud:google-cloud-spanner 6.111.1` → [Updates: `6.113.0`]
 - `net.bytebuddy:byte-buddy 1.18.5` → [Updates: `1.18.7-jdk5`]
 - `net.bytebuddy:byte-buddy-agent 1.18.5` → [Updates: `1.18.7-jdk5`]
 - `junit:junit 4.13.2`
 - `com.spotify.fmt:fmt-maven-plugin 2.29`

</details>

<details><summary>samples/spring-data-mybatis/postgresql/pom.xml (12)</summary>

 - `org.springframework.boot:spring-boot-starter-parent 3.5.11` → [Updates: `3.5.13`, `4.0.5`]
 - `org.springframework.data:spring-data-bom 2025.0.5` → [Updates: `2025.1.4`]
 - `com.google.cloud:libraries-bom 26.76.0` → [Updates: `26.78.0`]
 - `org.testcontainers:testcontainers-bom 2.0.3` → [Updates: `2.0.4`]
 - `org.mybatis.spring.boot:mybatis-spring-boot-starter 3.0.5` → [Updates: `4.0.1`]
 - `org.mybatis.dynamic-sql:mybatis-dynamic-sql 1.5.2` → [Updates: `2.0.0`]
 - `org.postgresql:postgresql 42.7.10`
 - `com.google.collections:google-collections 1.0`
 - `net.bytebuddy:byte-buddy 1.18.5` → [Updates: `1.18.7-jdk5`]
 - `net.bytebuddy:byte-buddy-agent 1.18.5` → [Updates: `1.18.7-jdk5`]
 - `junit:junit 4.13.2`
 - `com.spotify.fmt:fmt-maven-plugin 2.29`

</details>

</blockquote>
</details>

<details><summary>regex (4)</summary>
<blockquote>

<details><summary>.github/workflows/unmanaged_dependency_check.yaml (1)</summary>

 - `com.google.cloud:sdk-platform-java-config 3.57.0` → [Updates: `3.58.0`]

</details>

<details><summary>.kokoro/presubmit/graalvm-native-a.cfg (1)</summary>

 - `com.google.cloud:sdk-platform-java-config 3.57.0` → [Updates: `3.58.0`]

</details>

<details><summary>.kokoro/presubmit/graalvm-native-b.cfg (1)</summary>

 - `com.google.cloud:sdk-platform-java-config 3.57.0` → [Updates: `3.58.0`]

</details>

<details><summary>.kokoro/presubmit/graalvm-native-c.cfg (1)</summary>

 - `com.google.cloud:sdk-platform-java-config 3.57.0` → [Updates: `3.58.0`]

</details>

</blockquote>
</details>

---

- [ ] Check this box to trigger a request for Renovate to run again on this repository

**Comment by suztomo** [comment]
Dependency Dashboard from java-spanner-jdbc is no longer needed.
